scripts/buildArangoDB6.fish

```diff
@@ -68,7 +68,7 @@ if test "$SAN" = "On"
   set -xg CXX_NAME clang++
   # Suppress leaks detection only during building
   set -gx SAN_OPTIONS "detect_leaks=0"
-  set -l SANITIZERS "-fsanitize=address -fsanitize=undefined -fsanitize=float-divide-by-zero -fsanitize=leak"
+  set -l SANITIZERS "-fsanitize=address -fsanitize=undefined -fsanitize=float-divide-by-zero -fsanitize=leak -fsanitize-address-use-after-return=never"
```

Comment:
detect_stack_use_after_return=false
We disable it in runtime.
Better to disable in compile time
https://github.com/arangodb/oskar/blob/master/scripts/lib/tests.fish#L32


https://clang.llvm.org/docs/AddressSanitizer.html